README.md

```diff
@@ -1,94 +1,17 @@
 # Caliopen Messaging Platform
 
-This repository hosts all the code source, the scripts and the tools for the CaliOpen project.
+> The best way to discuss with your contacts, while preserving your privacy.
 
-All issues related to this project must be created in this [repository](https://github.com/CaliOpen/Caliopen/issues).
+![Actual POC](doc/assets/play_with_poc.png)
 
+This repository hosts all the code source, the scripts and the tools for the [CaliOpen](https://caliopen.org) project.
 
-# Repositories architecture
```

Comment:
I'm ok to move repository structure description in doc/for-developers but we must give links to this documentation (devtools/README and doc/for-developers with an index link) in this top README